src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReport.java

```diff
@@ -109,17 +111,18 @@ private void initReportSummary2HdrList(List<HearingsByHearingTypeReportSummary2H
     }
 
     private String getSubSplitSitAlone(HearingTypeItem hearingTypeItem) {
-        if ("Full".equals(hearingTypeItem.getValue().getHearingSitAlone())) {
-            return "Full Panel";
-        } else if (YES.equals(hearingTypeItem.getValue().getHearingSitAlone())) {
-            return "EJ Sit Alone";
-        } else {
-            return "";
+        if (!Strings.isNullOrEmpty(hearingTypeItem.getValue().getHearingSitAlone()))  {
```

Comment:
Here consider using StringUtils.isNotBlank after import org.apache.commons.lang3.StringUtils. It reads easier.